### ISSUE TITLE
Graphviz executables not found

### ISSUE THREAD

**Issue by BasileMarquier** [chrisalbon/notes#30]
Hello,

  I cannot have the graph because Python does not recognize Graphviz executables. I installed graphviz in anaconda but it still does not work. I do not know how to fix this.

Best wishes,

**Comment by DanielBeddows** [comment]
I had a few problems initially also, but resolved after installing pydotplus and graphviz separately. Apologies I can't remember exactly what I did. What is the exact error message you get?

**Comment by underground-11** [comment]
It was `InvocationException: GraphViz's executables not found` @DanielBeddows

**Comment by DanielBeddows** [comment]
Yep, that is it. I'm using Ubuntu, but this is good enough explanation here (link below) A similar approach should fix it if you are using windows, if using Linux I think for some reason sudo pip3 install graphviz did the trick. (using pip3, rather than sudo apt-get install python3-graphviz)... noting I am using python 3.4

https://stackoverflow.com/questions/18438997/why-is-pydot-unable-to-find-graphvizs-executables-in-windows-8

**Comment by BasileMarquier** [comment]
Hello guys,

  Many thanks for your help. I managed to do it... I downloaded a zip file and had to specify in two lines the Path. Then it worked.

Thanks,
